src/pool/srv_target.c

```diff
@@ -1887,6 +1887,7 @@ ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 			/* Ignore DTX resync failure that is not fatal. */
 			D_WARN("dtx_resync_ult failure %d\n", ret);
 			D_FREE(arg);
+			goto out;
```

Comment:
Seems unnecessary.

Reply:
hmm,  I was not intended to change it, maybe some failure when rebasing, I'll remove it.